runtime-sdk/modules/contracts/src/types.rs

```diff
@@ -217,6 +217,20 @@ pub struct InstanceStorageQueryResult {
     pub value: Option<Vec<u8>>,
 }
 
+/// Instance raw storage query.
+#[derive(Clone, Debug, cbor::Encode, cbor::Decode)]
+pub struct InstanceRawStorageQuery {
+    /// Instance identifier.
+    pub id: InstanceId,
+}
+
+/// Instance raw storage query result.
+#[derive(Clone, Debug, cbor::Encode, cbor::Decode)]
+pub struct InstanceRawStorageQueryResult {
+    /// List of key -> CBOR encoded value fields of public contract state.
```

Comment:
Actually whether the value is CBOR-encoded is up to the contract. With our contract-sdk this is usually the case, but in general the contract could use something else.